dll/win32/shell32/wine/shellpath.c

```diff
@@ -106,6 +106,127 @@ DoGetProductType(PNT_PRODUCT_TYPE ProductType)
 	########## Combining and Constructing paths ##########
 */
 
+/* @implemented */
+static BOOL WINAPI
+PathSearchOnExtensionsW(LPWSTR path, LPCWSTR *dirs, BOOL flag, DWORD dwWhich)
```

Comment:
Instead of `flag`, give a more descriptive name.

Reply:
OK.